src/renderer/common/components/menu/MenuContent.tsx

```diff
@@ -29,55 +27,119 @@ interface IBaseProps {
+
         return ReactDOM.createPortal(
             (
-                <AccessibleMenu doBackFocusMenuButton={this.props.doBackFocusMenuButton} visible={open} toggleMenu={toggle}>
+                <AccessibleMenu
+                    doBackFocusMenuButton={() => this.props.menuButtonRef.current?.focus()}
+                    visible={true}
+                    toggleMenu={this.props.closeMenu}
+                >
                     <div
-                        style={this.props.menuStyle}
+                        style={this.state.menuStyle.content}
                         id={this.props.id}
-                        aria-hidden={!this.props.open}
+                        aria-hidden={false}
                         role="menu"
-                        aria-expanded={this.props.open}
-                        ref={(ref) => {
-                            if (setContentRef) {
-                                setContentRef(ref);
-                            }
-                            if (ref) {
-                                const firstButt = ref.querySelector("button");
-                                firstButt.focus();
-                            }
-                        }}
+                        aria-expanded={true}
+                        ref={this.accessibleMenuContentRef}
                     >
-                        {this.props.children}
+
+                        <div className={stylesDropDown.dropdown_menu}>
+                            {this.props.children}
+                            <div id={"arrow"} aria-hidden={true} ref={this.arrowMenuRef} style={this.state.menuStyle.arrow}></div>
+                        </div>
                     </div>
                 </AccessibleMenu>
             ),
             this.rootElement,
         );
     }
+
+    private loadContentStyles() {
+
+        const cleanup = autoUpdate(
+            this.props.menuButtonRef.current,
+            this.accessibleMenuContentRef.current,
+            () => computePosition(this.props.menuButtonRef.current, this.accessibleMenuContentRef.current, {
+                placement: "bottom",
+                middleware: [flip(), shift({ padding: 5 }), offset(10), arrow({ element: this.arrowMenuRef.current })],
+            })
+                .then(({ x, y, placement, middlewareData: { arrow: { x: arrowX, y: arrowY } } }) => {
+
+                    const staticSide = {
+                        top: "bottom",
+                        right: "left",
+                        bottom: "top", // not used
+                        left: "right", // not used
+                    }[placement.split("-")[0]];
+
+                    this.setState({
+                        menuStyle: {
+                            content: {
+                                position: "absolute",
+                                left: `${x}px`,
+                                top: `${y}px`,
+                                zIndex: "1",
```

Comment:
z-index was missing here. That solved the problem of the arrow not showing in the list view.